src/core/selinux-access.c

```diff
@@ -25,6 +26,8 @@
 #include "strv.h"
 
 static bool initialized = false;
+static uid_t manager_auid = UID_INVALID;
+static uid_t log_auid = UID_INVALID;
```

Comment:
gah, it's frickin ugly we have to maintain static variables to pass around the info here. meh.